WindowManager/app/src/main/java/com/example/windowmanagersample/SampleTools.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.windowmanagersample
+
+import android.graphics.Rect
+import android.view.View
+import android.widget.FrameLayout
+import androidx.window.DisplayFeature
+
+/**
+ * Get the bounds of the display feature translated to the View's coordinate space and current
+ * position in the window. This will also include view padding in the calculations.
+ */
+fun getFeaturePositionInViewRect(
+    displayFeature: DisplayFeature,
+    view: View,
+    includePadding: Boolean = true
+): Rect? {
```

Comment:
For these type of functions, you'd typical accept the Rect as a parm, to allow the caller to cache the rect.

Reply:
I'd return a boolean based on success: `getFeatureBoundsInWindow(..., result: Rect): Boolean` and return false if there's no intersection. This might be overkill for a sample.

Reply:
I'm going to resolve this for now. I think one of the concerns is that your displayFeatures could move/change depending on device configuration in the future, and I have to think more about the best way to check if features have moved or values have changed where you'd have to recompute this bounds and the best guidance around doing that.